### PR TITLE
Data model updates

### PR DESCRIPTION
**Pull request by eliascodes**
Some minor updates based on work on the data sync tool.

Some inconsistencies between the schema diagram and the migrations have also been fixed.

### CODE REVIEW

File: database/migrations/sql/20180604_06_create_users.sql [hunk 1]
```diff
@@ -12,13 +12,15 @@ CREATE TABLE gender (
...
+  user_password                    VARCHAR(64),
+  qr_code                          VARCHAR UNIQUE,
   gender_id                        INT NOT NULL,
-  email                            VARCHAR(100) NOT NULL UNIQUE,
+  email                            VARCHAR(100) UNIQUE,
   phone_number                     VARCHAR(20),
   post_code                        VARCHAR(10),
   birth_year                       INT,
+  is_email_confirmed               BOOLEAN NOT NULL DEFAULT false,
+  is_phone_number_confirmed        BOOLEAN NOT NULL DEFAULT false,
```
Comment (astroash) [inline database/migrations/sql/20180604_06_create_users.sql]: Are we going to be sending text message confirmations?

File: database/seeds/development/11_access_role.seed.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+exports.seed = (knex) =>
```
Comment (astroash) [inline database/seeds/development/11_access_role.seed.js]: Lol, I have this in my pr. I'm gunna have to rewrite all my test.

Comment (eliascodes) [reply database/seeds/development/11_access_role.seed.js]: Should just need to do find/replace, no?

File: database/migrations/sql/20180604_06_create_users.sql (hunk 1 above)
Comment (eliascodes) [reply database/migrations/sql/20180604_06_create_users.sql]: Maybe. Either way, we should know whether we know this, if you get me.

Comment (astroash) [reply database/migrations/sql/20180604_06_create_users.sql]: Nope, but it's not overly important right now.

**Pull request merged**